src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -181,8 +190,13 @@ class ArrayField extends React.Component {
    *   determined from path
    */
   handleSave(index, fieldName) {
-    const newEditingArray = _.set(index, false, this.state.editing);
-    this.setState({ editing: newEditingArray }, () => {
+    const { uiSchema, arrayData } = this.props;
+    // Prevent save if the `duplicateKey` option is set and there are duplicates
```

Comment:
Is...is this a good idea? Preventing submission makes good sense, but preventing saving makes me nervous. What's the reasoning behind this?

Reply:
That's bad wording... we're not really preventing saving here, just preventing the edit mode from switching to `false` and closing the array card.